helix-core/src/test/java/org/apache/helix/TestHelper.java

```diff
@@ -76,8 +79,10 @@
 
 public class TestHelper {
   private static final Logger LOG = LoggerFactory.getLogger(TestHelper.class);
-  public static final long WAIT_DURATION = 60 * 1000L; // 60 seconds
-  public static final int DEFAULT_REBALANCE_PROCESSING_WAIT_TIME = 1500;
+  public static final long WAIT_DURATION = 10 * 1000L; // 10 seconds
+  public static final long POLL_DURATION = 10L;        // 10 milli-seconds
+  public static final int DEFAULT_REBALANCE_PROCESSING_WAIT_TIME = 1000;
```

Comment:
Why we want to change this? Do we expect to fail it quicker?

Reply:
I have reverted this change to keep the original values for now.